recipes/json-cfg/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "json-cfg" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d3dd1ab30b16a3bb249b6eb35fcc42198f9656f33127e36a3fadb5e37f50d45b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+  run:
+    - kwonly-args >=1.0.7
+    - python >=3.4
```

Does this really not support Python 2?
```suggestion
    - python >=2.7
  run:
    - kwonly-args >=1.0.7
    - python >=2.7
```
It seems like a package that is backporting Python 3-like features so it would be odd, but it's fine if 3.4 really is the minimum

---

It does, from what I can [see](https://github.com/pasztorpisti/json-cfg/blob/0ccde013987910f2bb37a46788c0b8d3053a9ac6/.travis.yml#L3-L9), but since it's not supported anywhere anymore I wasn't sure if I should include it since I wasn't sure how this affects conda-forge.

---

Also, thanks for the review!

---

And I'm not 100% sure how I can specify `2.7` and `>= 3.4`.  Is it `python ==2.7, >=3.4`?

---

The classifiers suggest even [more](https://github.com/pasztorpisti/json-cfg/blob/0ccde013987910f2bb37a46788c0b8d3053a9ac6/setup.py#L54-L61) but I'm not 100% sure if that means that they support every language version, since it just mentions generic python 2 and 3 versions?
The package hasn't been maintained for a long time so I don't think that I can get any feedback from them, unfortunately. I'm just adding this because it's a dependency of another recipe that I would like to use. But sorry for the extra work, should have investigated that more.

---

Last thing to add 😄 
This package depends on another package from the same author which also doesn't list any specific versions (https://github.com/pasztorpisti/kwonly-args, I'm the maintainer of its feedstock as well). 
Does that mean that I can also add `python >=2.4`(Which is the lowest I can find in the [releases](https://github.com/pasztorpisti/kwonly-args/releases/tag/v1.0.9)) to the kwonly-args [feedstock](https://github.com/conda-forge/kwonly-args-feedstock) as well? I was hesitant because of the bot part mentioned above but then I'd add it there as well.

---

> It does, from what I can see, but since it's not supported anywhere anymore I wasn't sure if I should include it since I wasn't sure how this affects conda-forge. (Because of https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-python-2-7, or does that do something different?)

This only applies to packages which can't be `noarch: python`.

> And I'm not 100% sure how I can specify 2.7 and >= 3.4. Is it python ==2.7, >=3.4?

I think it's `python =2.7|>=3.4` (the comma means "and").

> Does that mean that I can also add python >=2.4

Yup that's fine. Conda-forge has never supported versions that old so in reality it's hard to use versions before 2.7 (though personally I run some tests with Python 2.6 and I doubt I'm the only one).

---

Ahh ok, thanks for the clarification and help. Then I'll change everything accordingly.